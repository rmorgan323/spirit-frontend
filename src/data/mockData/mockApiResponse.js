const mockApiResponse = {
  status: 'success'
};

export default mockApiResponse;