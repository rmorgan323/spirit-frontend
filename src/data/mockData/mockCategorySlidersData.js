/*eslint-disable camelcase*/

export const mockTitle = 'Capacities';

export const mockSlidersArray = [
  'Attention',
  'Action',
  'Affect',
  'Autonomic',
  'Arousability'
];

export const mockDatabaseNamesArray = [
  'mod_2_attention',
  'mod_2_action',
  'mod_2_affect',
  'mod_2_autonomic',
  'mod_2_arousability'
];

export const mockReminderAsterisk = [null, '*', '*', '*', '*'];

export const mockReminder = '* If selected, remember to make a hypothesized arousal chart.';

export const mockSliderTitle = 'Attention';

export const mockDatabaseName = 'mod_2_attention';

