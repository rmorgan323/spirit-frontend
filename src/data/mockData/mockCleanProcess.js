/*eslint-disable camelcase*/

export const mockCleanProcess = {
  id: 10,
  sen_h_vestibular: null,
  sen_h_proprioception: null,
  sen_h_tactile: null,
  sen_h_auditory: null,
  sen_h_visual: null,
  sen_h_intero: null,
  sen_1_ability: null,
  sen_1_match: null,
  sen_1_classification: null,
  sen_1_spatial: null,
  sen_1_figure: null,
  sen_1_inferences: null,
  sen_2_initiation: null,
  sen_2_imitation: null,
  sen_2_affordance: null,
  sen_2_generativity: null,
  sen_3_sequencing: null,
  sen_3_sequencing_gestalt: null,
  sen_3_schema_selection: null,
  sen_3_schema_expansion: null,
  sen_3_modification: null,
  sen_4_timing: null,
  sen_4_timing_force: null,
  sen_4_spatial: null,
  sen_4_temporal: null,
  sen_4_control: null,
  sen_4_monitoring: null,
  sen_5_oral: null,
  sen_5_ocular: null,
  sen_5_postural: null,
  sen_5_limb: null,
  sen_5_gross: null,
  sen_5_fine: null,
  sen_5_tool: null,
  sen_5_projected: null,
  sen_6_somatodyspraxia: null,
  sen_6_visual: null,
  sen_6_generalized: null,
  sen_6_bmc: null,
  sen_6_praxis: null,
  mod_h_vestibular: null,
  mod_h_proprioception: null,
  mod_h_tactile: null,
  mod_h_auditory: null,
  mod_h_visual: null,
  mod_h_intero: null,
  mod_1_defensiveness: null,
  mod_1_over: null,
  mod_1_under: null,
  mod_1_variable: null,
  mod_2_attention: null,
  mod_2_action: null,
  mod_2_affect: null,
  mod_2_autonomic: null,
  mod_3_physiological: null,
  mod_3_sensory: null,
  mod_3_motor: null,
  mod_3_emotional: null,
  mod_3_cognitive: null,
  mod_3_social: null,
  mod_3_prosocial: null,
  mod_4a_passive: null,
  mod_4a_co_regulation: null,
  mod_4a_approach: null,
  mod_4a_adaptive: null,
  mod_4b_active: null,
  mod_4b_self_regulation: null,
  mod_4b_avoid: null,
  mod_4b_maladaptive: null,
  exe_1_orient: null,
  exe_1_inhibition: null,
  exe_2a_working: null,
  exe_2a_shifting: null,
  exe_2b_verbal: null,
  exe_2b_spatial: null,
  exe_2b_general: null,
  exe_2b_global: null,
  exe_2b_flexibility: null,
  exe_3_sequencing: null,
  exe_3_monitoring: null,
  exe_4a_future: null,
  exe_4a_spatial: null,
  exe_4a_problem: null,
  exe_4a_reflective: null,
  exe_4a_internalized: null,
  exe_4b_motivational: null,
  exe_4b_effortful: null,
  exe_4b_self_control: null,
  exe_4b_self_regulation: null,
  exe_4b_self_compassion: null,
  pos_h_vestibular: null,
  pos_h_proprioception: null,
  pos_h_tactile: null,
  pos_h_auditory: null,
  pos_h_visual: null,
  pos_h_intero: null,
  pos_2_stability: null,
  pos_2_mobility: null,
  pos_2_tonic: null,
  pos_2_phasic: null,
  pos_2_muscle: null,
  pos_3_sagittal: null,
  pos_3_frontal: null,
  pos_3_transverse: null,
  pos_3_midlines: null,
  pos_5_suck: null,
  pos_5_ocular: null,
  pos_5_antigravity: null,
  pos_5_reflex: null,
  pos_5_alignment_COG: null,
  pos_5_alignment_orient: null,
  pos_5_anticipatory: null,
  pos_5_interactive: null,
  pos_6_push: null,
  pos_6_bilateral: null,
  pos_6_grasp: null,
  pos_6_strength: null,
  pos_6_effortful: null,
  soc_1_regulation: null,
  soc_1_engagement: null,
  soc_1_reciprocity: null,
  soc_1_problem: null,
  soc_1_representation: null,
  soc_1_triangular: null,
  soc_1_comparative: null,
  soc_1_reflective: null,
  soc_2_self_awareness: null,
  soc_2_joint: null,
  soc_2_social_referencing: null,
  soc_2_imitation: null,
  soc_2_inferential: null,
  soc_2_contingent: null,
  soc_2_theory: null,
  soc_2_central: null,
  soc_2_mentalizing: null,
  soc_2_empathy: null,
  soc_2_social_motivators: null,
  soc_2_pro_social: null,
  soc_3_affective: null,
  soc_3_emotional_contagion: null,
  soc_3_activation_low: null,
  soc_3_emotional_cascade: null,
  soc_3_activation_high: null,
  soc_3_emotional_differentiation: null,
  soc_3_emotional_association: null,
  soc_3_self_control: null,
  soc_3_self_regulation: null,
  soc_3_self_compassion: null,
  soc_4_mental: null,
  soc_4_thoughts: null,
  soc_4_attachments: null,
  soc_4_trauma: null,
  session_id: 10,
  created_at: "2018-02-20T21:25:45.454Z",
  updated_at: "2018-02-20T21:25:45.454Z",
  mod_2_arousability: null,
  exe_3_initiation: null,
  exe_3_planning: null,
  pos_4_core: null,
  pos_4_shoulder: null,
  pos_4_pelvic: null,
  pos_4_head: null,
  pos_4_eyes: null,
  pos_4_hand: null,
  pos_4_lower: null,
  pos_4_foot: null,
  soc_2_social_perspective: null,
  pos_1_outer: null,
  pos_1_inner: null,
  exe_4a_generativity: null
};
