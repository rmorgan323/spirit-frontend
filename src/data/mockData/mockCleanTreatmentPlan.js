/*eslint-disable camelcase*/

export const mockCleanTreatmentPlan = {
  id: 10,
  modulation_sensory: null,
  modulation_task: null,
  modulation_environment: null,
  modulation_predictability: null,
  modulation_self_regulation: null,
  modulation_interaction: null,
  modulation_JRC_AR_notes: null,
  posture_sensory: null,
  posture_task: null,
  posture_environment: null,
  posture_predictability: null,
  posture_self_regulation: null,
  posture_interaction: null,
  posture_JRC_AR_notes: null,
  sensory_discrimination_sensory: null,
  sensory_discrimination_task: null,
  sensory_discrimination_environment: null,
  sensory_discrimination_predictability: null,
  sensory_discrimination_self_regulation: null,
  sensory_discrimination_interaction: null,
  sensory_discrimination_JRC_AR_notes: null,
  social_emotional_sensory: null,
  social_emotional_task: null,
  social_emotional_environment: null,
  social_emotional_predictability: null,
  social_emotional_self_regulation: null,
  social_emotional_interaction: null,
  social_emotional_JRC_AR_notes: null,
  executive_functioning_sensory: null,
  executive_functioning_task: null,
  executive_functioning_environment: null,
  executive_functioning_predictability: null,
  executive_functioning_self_regulation: null,
  executive_functioning_interaction: null,
  executive_functioning_JRC_AR_notes: null,
  session_id: 10,
  created_at: "2018-02-20T21:25:45.468Z",
  updated_at: "2018-02-20T21:25:45.468Z"
};
