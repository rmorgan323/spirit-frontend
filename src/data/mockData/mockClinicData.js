/*eslint-disable camelcase*/

export const mockClinicData = {
  name: 'I Need Therapy',
  abbreviation: 'INT',
  clinic_id: 10,
  passcode: 'f83jds8e'
};
