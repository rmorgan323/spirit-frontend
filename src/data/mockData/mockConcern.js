/*eslint-disable camelcase*/

export const mockConcern = {
  id: 5,
  description: "Concern 1",
  domain_1: true,
  domain_2: true,
  domain_3: false,
  domain_4: true,
  domain_5: false,
  domain_6: false,
  notes: "Some notes.",
  patient_id: 5,
  created_at: "2018-02-20T20:08:19.142Z",
  updated_at: "2018-02-20T20:08:19.142Z"
};
