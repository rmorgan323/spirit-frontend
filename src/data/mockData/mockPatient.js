/*eslint-disable camelcase*/

export const mockPatient = {
  id: 5,
  abstracted_name: "INTrm5",
  clinic_name: "I Need Therapy",
  ot_id: 1,
  created_at: "2018-02-20T20:07:54.509Z",
  updated_at: "2018-02-20T20:07:54.509Z"
};
