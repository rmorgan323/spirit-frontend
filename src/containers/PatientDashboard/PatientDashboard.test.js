import React from 'react';
import {
  PatientDashboard,
  mapStateToProps,
  mapDispatchToProps
} from './PatientDashboard';
import { shallow } from 'enzyme';
import { mockPatient } from '../../data/mockData/mockPatient';
import { mockConcerns } from '../../data/mockData/mockConcern';

describe('PatientDashboard tests', () => {
  let addConcern;
  let currentPatient;
  let wipeStoreFromPatientDashboard;
  let renderedPatientDashboard;

  beforeEach(() => {
    addConcern = jest.fn();
    currentPatient = mockPatient;
    wipeStoreFromPatientDashboard = jest.fn();

    renderedPatientDashboard = shallow(
      <PatientDashboard
        addConcern={addConcern}
        currentPatient={currentPatient}
        wipeStoreFromPatientDashboard={wipeStoreFromPatientDashboard}
      />
    );
  });

  it('Should match the snapshot', () => {
    expect(renderedPatientDashboard).toMatchSnapshot();
  });

  it('should set state when toggleDomain is called', () => {
    expect(renderedPatientDashboard.state('domain1')).toEqual(false);

    renderedPatientDashboard.instance().toggleDomain('domain1');

    expect(renderedPatientDashboard.state('domain1')).toEqual(true);

    renderedPatientDashboard.instance().toggleDomain('domain1');

    expect(renderedPatientDashboard.state('domain1')).toEqual(false);
  });

  it('should set state when handleChange is called', () => {
    const mockEvent = {
      target: { name: 'concernInput', value: 'New Concern'}
    };
    expect(renderedPatientDashboard.state('concernInput')).toEqual('');

    renderedPatientDashboard.instance().handleChange(mockEvent);

    expect(renderedPatientDashboard.state('concernInput')).toEqual('New Concern');
  });

  it('should reset state when clearInputs is called', () => {
    const mockEvent = {
      target: { name: 'notesInput', value: 'Notes'}
    };
    renderedPatientDashboard.instance().toggleDomain('domain2');
    renderedPatientDashboard.instance().handleChange(mockEvent);

    expect(renderedPatientDashboard.state('domain2')).toEqual(true);
    expect(renderedPatientDashboard.state('notesInput')).toEqual('Notes');

    renderedPatientDashboard.instance().clearInputs();

    expect(renderedPatientDashboard.state('domain2')).toEqual(false);
    expect(renderedPatientDashboard.state('notesInput')).toEqual('');
  });
});

describe('mapStateToProps tests', () => {
  it('Should pull patientConcerns and currentPatient from store', () => {
    const patientConcerns = [mockConcerns];
    const currentPatient = mockPatient;
    const mockStore = { patientConcerns, currentPatient };
    const result = mapStateToProps(mockStore);

    expect(result.patientConcerns).toEqual(mockStore.patientConcerns);
    expect(result.currentPatient).toEqual(mockStore.currentPatient);
  });
});

describe('mapDispatchToProps tests', () => {
  it('Should call dispatch when addConcern is called', () => {
    const mockDispatch = jest.fn();
    const mockParameters = [5, mockConcerns];
    const result = mapDispatchToProps(mockDispatch);

    result.addConcern(...mockParameters);
    expect(mockDispatch).toHaveBeenCalled();
  });

  it('Should call dispatch when wipeStoreFromPatientDashboard is called', () => {
    const mockDispatch = jest.fn();
    const result = mapDispatchToProps(mockDispatch);

    result.addConcern();
    expect(mockDispatch).toHaveBeenCalled();
  });
});
