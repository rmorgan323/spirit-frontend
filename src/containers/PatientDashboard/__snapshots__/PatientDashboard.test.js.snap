// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PatientDashboard tests Should match the snapshot 1`] = `
ShallowWrapper {
  "length": 1,
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <PatientDashboard
    addConcern={[Function]}
    currentPatient={
      Object {
        "abstracted_name": "INTrm5",
        "clinic_name": "I Need Therapy",
        "created_at": "2018-02-20T20:07:54.509Z",
        "id": 5,
        "ot_id": 1,
        "updated_at": "2018-02-20T20:07:54.509Z",
      }
    }
    wipeStoreFromPatientDashboard={[Function]}
  />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": Array [
        <div
          className="pd-header-row"
        >
          <h3
            className="pd-first"
          >
            Primary Concerns
          </h3>
          <h3
            className="pd-second"
          >
            Domains
          </h3>
          <h3
            className="pd-third"
          >
            Notes
          </h3>
        </div>,
        <Connect(Component) />,
        <div
          className="pd-input-row"
        >
          <input
            className="primary-concern"
            name="concernInput"
            onChange={[Function]}
            placeholder="Add Concern"
            value=""
          />
          <p>
            <span
              className="domains"
              onClick={[Function]}
            >
              1
            </span>
            <span
              className="domains"
              onClick={[Function]}
            >
              2
            </span>
            <span
              className="domains"
              onClick={[Function]}
            >
              3
            </span>
            <span
              className="domains"
              onClick={[Function]}
            >
              4
            </span>
            <span
              className="domains"
              onClick={[Function]}
            >
              5
            </span>
            <span
              className="domains"
              onClick={[Function]}
            >
              6
            </span>
          </p>
          <input
            className="primary-concern-notes"
            name="notesInput"
            onChange={[Function]}
            placeholder="Add Notes"
            value=""
          />
          <button
            className="save-primary-concern"
            onClick={[Function]}
          >
            save
          </button>
        </div>,
        <div
          className="sam-holder"
        >
          <img
            alt="sam-big"
            className="sam-image"
            src="/assets/sam-new.svg"
          />
          <div
            className="sam-dot-1 sam-dots"
          >
            1
          </div>
          <div
            className="sam-dot-2 sam-dots"
          >
            2
          </div>
          <div
            className="sam-dot-3 sam-dots"
          >
            3
          </div>
          <div
            className="sam-dot-4 sam-dots"
          >
            4
          </div>
          <div
            className="sam-dot-5 sam-dots"
          >
            5
          </div>
          <div
            className="sam-dot-6 sam-dots"
          >
            6
          </div>
        </div>,
      ],
      "className": "PatientDashboard",
    },
    "ref": null,
    "rendered": Array [
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": Array [
            <h3
              className="pd-first"
            >
              Primary Concerns
            </h3>,
            <h3
              className="pd-second"
            >
              Domains
            </h3>,
            <h3
              className="pd-third"
            >
              Notes
            </h3>,
          ],
          "className": "pd-header-row",
        },
        "ref": null,
        "rendered": Array [
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": "Primary Concerns",
              "className": "pd-first",
            },
            "ref": null,
            "rendered": "Primary Concerns",
            "type": "h3",
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": "Domains",
              "className": "pd-second",
            },
            "ref": null,
            "rendered": "Domains",
            "type": "h3",
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": "Notes",
              "className": "pd-third",
            },
            "ref": null,
            "rendered": "Notes",
            "type": "h3",
          },
        ],
        "type": "div",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "class",
        "props": Object {},
        "ref": null,
        "rendered": null,
        "type": [Function],
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": Array [
            <input
              className="primary-concern"
              name="concernInput"
              onChange={[Function]}
              placeholder="Add Concern"
              value=""
            />,
            <p>
              <span
                className="domains"
                onClick={[Function]}
              >
                1
              </span>
              <span
                className="domains"
                onClick={[Function]}
              >
                2
              </span>
              <span
                className="domains"
                onClick={[Function]}
              >
                3
              </span>
              <span
                className="domains"
                onClick={[Function]}
              >
                4
              </span>
              <span
                className="domains"
                onClick={[Function]}
              >
                5
              </span>
              <span
                className="domains"
                onClick={[Function]}
              >
                6
              </span>
            </p>,
            <input
              className="primary-concern-notes"
              name="notesInput"
              onChange={[Function]}
              placeholder="Add Notes"
              value=""
            />,
            <button
              className="save-primary-concern"
              onClick={[Function]}
            >
              save
            </button>,
          ],
          "className": "pd-input-row",
        },
        "ref": null,
        "rendered": Array [
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "className": "primary-concern",
              "name": "concernInput",
              "onChange": [Function],
              "placeholder": "Add Concern",
              "value": "",
            },
            "ref": null,
            "rendered": null,
            "type": "input",
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": Array [
                <span
                  className="domains"
                  onClick={[Function]}
                >
                  1
                </span>,
                <span
                  className="domains"
                  onClick={[Function]}
                >
                  2
                </span>,
                <span
                  className="domains"
                  onClick={[Function]}
                >
                  3
                </span>,
                <span
                  className="domains"
                  onClick={[Function]}
                >
                  4
                </span>,
                <span
                  className="domains"
                  onClick={[Function]}
                >
                  5
                </span>,
                <span
                  className="domains"
                  onClick={[Function]}
                >
                  6
                </span>,
              ],
            },
            "ref": null,
            "rendered": Array [
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {
                  "children": "1",
                  "className": "domains",
                  "onClick": [Function],
                },
                "ref": null,
                "rendered": "1",
                "type": "span",
              },
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {
                  "children": "2",
                  "className": "domains",
                  "onClick": [Function],
                },
                "ref": null,
                "rendered": "2",
                "type": "span",
              },
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {
                  "children": "3",
                  "className": "domains",
                  "onClick": [Function],
                },
                "ref": null,
                "rendered": "3",
                "type": "span",
              },
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {
                  "children": "4",
                  "className": "domains",
                  "onClick": [Function],
                },
                "ref": null,
                "rendered": "4",
                "type": "span",
              },
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {
                  "children": "5",
                  "className": "domains",
                  "onClick": [Function],
                },
                "ref": null,
                "rendered": "5",
                "type": "span",
              },
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {
                  "children": "6",
                  "className": "domains",
                  "onClick": [Function],
                },
                "ref": null,
                "rendered": "6",
                "type": "span",
              },
            ],
            "type": "p",
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "className": "primary-concern-notes",
              "name": "notesInput",
              "onChange": [Function],
              "placeholder": "Add Notes",
              "value": "",
            },
            "ref": null,
            "rendered": null,
            "type": "input",
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": "save",
              "className": "save-primary-concern",
              "onClick": [Function],
            },
            "ref": null,
            "rendered": "save",
            "type": "button",
          },
        ],
        "type": "div",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": Array [
            <img
              alt="sam-big"
              className="sam-image"
              src="/assets/sam-new.svg"
            />,
            <div
              className="sam-dot-1 sam-dots"
            >
              1
            </div>,
            <div
              className="sam-dot-2 sam-dots"
            >
              2
            </div>,
            <div
              className="sam-dot-3 sam-dots"
            >
              3
            </div>,
            <div
              className="sam-dot-4 sam-dots"
            >
              4
            </div>,
            <div
              className="sam-dot-5 sam-dots"
            >
              5
            </div>,
            <div
              className="sam-dot-6 sam-dots"
            >
              6
            </div>,
          ],
          "className": "sam-holder",
        },
        "ref": null,
        "rendered": Array [
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "alt": "sam-big",
              "className": "sam-image",
              "src": "/assets/sam-new.svg",
            },
            "ref": null,
            "rendered": null,
            "type": "img",
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": "1",
              "className": "sam-dot-1 sam-dots",
            },
            "ref": null,
            "rendered": "1",
            "type": "div",
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": "2",
              "className": "sam-dot-2 sam-dots",
            },
            "ref": null,
            "rendered": "2",
            "type": "div",
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": "3",
              "className": "sam-dot-3 sam-dots",
            },
            "ref": null,
            "rendered": "3",
            "type": "div",
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": "4",
              "className": "sam-dot-4 sam-dots",
            },
            "ref": null,
            "rendered": "4",
            "type": "div",
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": "5",
              "className": "sam-dot-5 sam-dots",
            },
            "ref": null,
            "rendered": "5",
            "type": "div",
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": "6",
              "className": "sam-dot-6 sam-dots",
            },
            "ref": null,
            "rendered": "6",
            "type": "div",
          },
        ],
        "type": "div",
      },
    ],
    "type": "div",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": Array [
          <div
            className="pd-header-row"
          >
            <h3
              className="pd-first"
            >
              Primary Concerns
            </h3>
            <h3
              className="pd-second"
            >
              Domains
            </h3>
            <h3
              className="pd-third"
            >
              Notes
            </h3>
          </div>,
          <Connect(Component) />,
          <div
            className="pd-input-row"
          >
            <input
              className="primary-concern"
              name="concernInput"
              onChange={[Function]}
              placeholder="Add Concern"
              value=""
            />
            <p>
              <span
                className="domains"
                onClick={[Function]}
              >
                1
              </span>
              <span
                className="domains"
                onClick={[Function]}
              >
                2
              </span>
              <span
                className="domains"
                onClick={[Function]}
              >
                3
              </span>
              <span
                className="domains"
                onClick={[Function]}
              >
                4
              </span>
              <span
                className="domains"
                onClick={[Function]}
              >
                5
              </span>
              <span
                className="domains"
                onClick={[Function]}
              >
                6
              </span>
            </p>
            <input
              className="primary-concern-notes"
              name="notesInput"
              onChange={[Function]}
              placeholder="Add Notes"
              value=""
            />
            <button
              className="save-primary-concern"
              onClick={[Function]}
            >
              save
            </button>
          </div>,
          <div
            className="sam-holder"
          >
            <img
              alt="sam-big"
              className="sam-image"
              src="/assets/sam-new.svg"
            />
            <div
              className="sam-dot-1 sam-dots"
            >
              1
            </div>
            <div
              className="sam-dot-2 sam-dots"
            >
              2
            </div>
            <div
              className="sam-dot-3 sam-dots"
            >
              3
            </div>
            <div
              className="sam-dot-4 sam-dots"
            >
              4
            </div>
            <div
              className="sam-dot-5 sam-dots"
            >
              5
            </div>
            <div
              className="sam-dot-6 sam-dots"
            >
              6
            </div>
          </div>,
        ],
        "className": "PatientDashboard",
      },
      "ref": null,
      "rendered": Array [
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": Array [
              <h3
                className="pd-first"
              >
                Primary Concerns
              </h3>,
              <h3
                className="pd-second"
              >
                Domains
              </h3>,
              <h3
                className="pd-third"
              >
                Notes
              </h3>,
            ],
            "className": "pd-header-row",
          },
          "ref": null,
          "rendered": Array [
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": "Primary Concerns",
                "className": "pd-first",
              },
              "ref": null,
              "rendered": "Primary Concerns",
              "type": "h3",
            },
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": "Domains",
                "className": "pd-second",
              },
              "ref": null,
              "rendered": "Domains",
              "type": "h3",
            },
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": "Notes",
                "className": "pd-third",
              },
              "ref": null,
              "rendered": "Notes",
              "type": "h3",
            },
          ],
          "type": "div",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "class",
          "props": Object {},
          "ref": null,
          "rendered": null,
          "type": [Function],
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": Array [
              <input
                className="primary-concern"
                name="concernInput"
                onChange={[Function]}
                placeholder="Add Concern"
                value=""
              />,
              <p>
                <span
                  className="domains"
                  onClick={[Function]}
                >
                  1
                </span>
                <span
                  className="domains"
                  onClick={[Function]}
                >
                  2
                </span>
                <span
                  className="domains"
                  onClick={[Function]}
                >
                  3
                </span>
                <span
                  className="domains"
                  onClick={[Function]}
                >
                  4
                </span>
                <span
                  className="domains"
                  onClick={[Function]}
                >
                  5
                </span>
                <span
                  className="domains"
                  onClick={[Function]}
                >
                  6
                </span>
              </p>,
              <input
                className="primary-concern-notes"
                name="notesInput"
                onChange={[Function]}
                placeholder="Add Notes"
                value=""
              />,
              <button
                className="save-primary-concern"
                onClick={[Function]}
              >
                save
              </button>,
            ],
            "className": "pd-input-row",
          },
          "ref": null,
          "rendered": Array [
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "className": "primary-concern",
                "name": "concernInput",
                "onChange": [Function],
                "placeholder": "Add Concern",
                "value": "",
              },
              "ref": null,
              "rendered": null,
              "type": "input",
            },
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": Array [
                  <span
                    className="domains"
                    onClick={[Function]}
                  >
                    1
                  </span>,
                  <span
                    className="domains"
                    onClick={[Function]}
                  >
                    2
                  </span>,
                  <span
                    className="domains"
                    onClick={[Function]}
                  >
                    3
                  </span>,
                  <span
                    className="domains"
                    onClick={[Function]}
                  >
                    4
                  </span>,
                  <span
                    className="domains"
                    onClick={[Function]}
                  >
                    5
                  </span>,
                  <span
                    className="domains"
                    onClick={[Function]}
                  >
                    6
                  </span>,
                ],
              },
              "ref": null,
              "rendered": Array [
                Object {
                  "instance": null,
                  "key": undefined,
                  "nodeType": "host",
                  "props": Object {
                    "children": "1",
                    "className": "domains",
                    "onClick": [Function],
                  },
                  "ref": null,
                  "rendered": "1",
                  "type": "span",
                },
                Object {
                  "instance": null,
                  "key": undefined,
                  "nodeType": "host",
                  "props": Object {
                    "children": "2",
                    "className": "domains",
                    "onClick": [Function],
                  },
                  "ref": null,
                  "rendered": "2",
                  "type": "span",
                },
                Object {
                  "instance": null,
                  "key": undefined,
                  "nodeType": "host",
                  "props": Object {
                    "children": "3",
                    "className": "domains",
                    "onClick": [Function],
                  },
                  "ref": null,
                  "rendered": "3",
                  "type": "span",
                },
                Object {
                  "instance": null,
                  "key": undefined,
                  "nodeType": "host",
                  "props": Object {
                    "children": "4",
                    "className": "domains",
                    "onClick": [Function],
                  },
                  "ref": null,
                  "rendered": "4",
                  "type": "span",
                },
                Object {
                  "instance": null,
                  "key": undefined,
                  "nodeType": "host",
                  "props": Object {
                    "children": "5",
                    "className": "domains",
                    "onClick": [Function],
                  },
                  "ref": null,
                  "rendered": "5",
                  "type": "span",
                },
                Object {
                  "instance": null,
                  "key": undefined,
                  "nodeType": "host",
                  "props": Object {
                    "children": "6",
                    "className": "domains",
                    "onClick": [Function],
                  },
                  "ref": null,
                  "rendered": "6",
                  "type": "span",
                },
              ],
              "type": "p",
            },
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "className": "primary-concern-notes",
                "name": "notesInput",
                "onChange": [Function],
                "placeholder": "Add Notes",
                "value": "",
              },
              "ref": null,
              "rendered": null,
              "type": "input",
            },
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": "save",
                "className": "save-primary-concern",
                "onClick": [Function],
              },
              "ref": null,
              "rendered": "save",
              "type": "button",
            },
          ],
          "type": "div",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": Array [
              <img
                alt="sam-big"
                className="sam-image"
                src="/assets/sam-new.svg"
              />,
              <div
                className="sam-dot-1 sam-dots"
              >
                1
              </div>,
              <div
                className="sam-dot-2 sam-dots"
              >
                2
              </div>,
              <div
                className="sam-dot-3 sam-dots"
              >
                3
              </div>,
              <div
                className="sam-dot-4 sam-dots"
              >
                4
              </div>,
              <div
                className="sam-dot-5 sam-dots"
              >
                5
              </div>,
              <div
                className="sam-dot-6 sam-dots"
              >
                6
              </div>,
            ],
            "className": "sam-holder",
          },
          "ref": null,
          "rendered": Array [
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "alt": "sam-big",
                "className": "sam-image",
                "src": "/assets/sam-new.svg",
              },
              "ref": null,
              "rendered": null,
              "type": "img",
            },
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": "1",
                "className": "sam-dot-1 sam-dots",
              },
              "ref": null,
              "rendered": "1",
              "type": "div",
            },
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": "2",
                "className": "sam-dot-2 sam-dots",
              },
              "ref": null,
              "rendered": "2",
              "type": "div",
            },
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": "3",
                "className": "sam-dot-3 sam-dots",
              },
              "ref": null,
              "rendered": "3",
              "type": "div",
            },
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": "4",
                "className": "sam-dot-4 sam-dots",
              },
              "ref": null,
              "rendered": "4",
              "type": "div",
            },
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": "5",
                "className": "sam-dot-5 sam-dots",
              },
              "ref": null,
              "rendered": "5",
              "type": "div",
            },
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": "6",
                "className": "sam-dot-6 sam-dots",
              },
              "ref": null,
              "rendered": "6",
              "type": "div",
            },
          ],
          "type": "div",
        },
      ],
      "type": "div",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
      },
    },
  },
}
`;
